### PR TITLE
DOC-468 | BTS-1341 | ArangoSearch Views building in progress query warning

### PR DESCRIPTION
**Pull request by nerpaula**

### CODE REVIEW

File: 3.11/arangosearch.md [hunk 1]
```diff
@@ -134,6 +134,14 @@ logical and comparison operators, as well as
    both, with `FILTER`s after `SEARCH`, in which case the filter criteria will
    be applied to the search results as a post-processing step.
 
+{% hint 'info' %}
+Note that if you link a collection to a View and execute a query against this
+View while it is still being indexed, you may not get complete results. In the
+case where a View is still being built and simultaneously used in a query, the
+query includes an `1240` warning message informing you that the `arangosearch`
```
Comment (Simran-B) [inline 3.11/arangosearch.md:141]:
```suggestion
query includes a warning message (code `1240`) informing you that the `arangosearch`
```

File: 3.11/release-notes-api-changes311.md [hunk 2]
```diff
@@ -139,6 +139,14 @@ value of `0` as the `numberOfShards`. They now return the actual number of
 shards. This value can be higher than the configured `numberOfShards` value of
 the graph due to internally used hidden collections.
 
+#### Cursor API
+
+When you link a collection to an `arangosearch` View and run an AQL query
+against this View while it is still being indexed, you now receive a query
+warning. This warning alerts you about potentially incomplete results obtained
```
Comment (Simran-B) [inline 3.11/release-notes-api-changes311.md:146]:
```suggestion
against this View while it is still being indexed, you now receive the query result
including a warning. This warning alerts you about potentially incomplete results obtained
```

File: 3.11/release-notes-api-changes311.md [hunk 3]
```diff
@@ -522,6 +530,16 @@ In case of success, they still return `true`.
 You can wrap calls to these methods with a `try { ... }` block to catch errors,
 for example, in _arangosh_ or in Foxx services.
 
+### Query methods
+
+When using the `db._query` method to execute an AQL query against an
```
Comment (Simran-B) [inline 3.11/release-notes-api-changes311.md:535]:
```suggestion
When you use the `db._query()` method to execute an AQL query against an
```

File: 3.11/release-notes-api-changes311.md [hunk 4]
```diff
@@ -522,6 +530,16 @@ In case of success, they still return `true`.
...
+`arangosearch` View while it is still in the process of being built,
+the query now includes a warning message that the result set may not be
+complete due to the ongoing building process of the View.
```
Comment (Simran-B) [inline 3.11/release-notes-api-changes311.md:538]:
```suggestion
complete due to the ongoing indexing process of the View.
```

File: 3.11/release-notes-api-changes311.md [hunk 5]
```diff
@@ -522,6 +530,16 @@ In case of success, they still return `true`.
 You can wrap calls to these methods with a `try { ... }` block to catch errors,
 for example, in _arangosh_ or in Foxx services.
 
+### Query methods
+
+When using the `db._query` method to execute an AQL query against an
+`arangosearch` View while it is still in the process of being built,
+the query now includes a warning message that the result set may not be
```
Comment (Simran-B) [inline 3.11/release-notes-api-changes311.md:537]:
```suggestion
the query now includes a warning message that the results may not be
```

File: 3.12/arangosearch.md [hunk 6]
```diff
@@ -134,6 +134,14 @@ logical and comparison operators, as well as
    both, with `FILTER`s after `SEARCH`, in which case the filter criteria will
    be applied to the search results as a post-processing step.
 
+{% hint 'info' %}
+Note that if you link a collection to a View and execute a query against this
+View while it is still being indexed, you may not get complete results. In the
+case where a View is still being built and simultaneously used in a query, the
+query includes an `1240` warning message informing you that the `arangosearch`
```
Comment (Simran-B) [inline 3.12/arangosearch.md:141]:
```suggestion
query includes a warning message (code `1240`) informing you that the `arangosearch`
```

File: 3.11/release-notes-api-changes311.md [hunk 7]
```diff
@@ -522,6 +530,16 @@ In case of success, they still return `true`.
 You can wrap calls to these methods with a `try { ... }` block to catch errors,
 for example, in _arangosh_ or in Foxx services.
 
+### Query methods
```
Comment (Simran-B) [inline 3.11/release-notes-api-changes311.md:533]:
```suggestion
### Query method
```

**Review by Simran-B — CHANGES_REQUESTED** [review]
@MBkkt Just to double check, with inverted indexes / search-alias Views, you never see incomplete results?

**Comment by MBkkt** [comment]
> Just to double check, with inverted indexes / search-alias Views, you never see incomplete results?

Yes, because index invisible for user while it's building

**Review by hkernbach — APPROVED** [review]

**Pull request merged**